src/main/kotlin/no/nav/syfo/aktivitetskrav/domain/AktivitetskravVarsel.kt

```diff
@@ -19,17 +19,23 @@ data class AktivitetskravVarsel internal constructor(
 
     companion object {
         fun create(
+            type: VarselType,
             document: List<DocumentComponentDTO>,
-            type: VarselType = VarselType.FORHANDSVARSEL_STANS_AV_SYKEPENGER,
-        ) =
-            AktivitetskravVarsel(
+        ): AktivitetskravVarsel {
+            if (document.isEmpty()) {
+                throw IllegalArgumentException("Varsel can't have empty document")
+            }
+
+            return AktivitetskravVarsel(
                 uuid = UUID.randomUUID(),
                 createdAt = nowUTC(),
                 journalpostId = null,
-                svarfrist = if (type == VarselType.FORHANDSVARSEL_STANS_AV_SYKEPENGER) LocalDate.now().plusWeeks(3) else null,
+                svarfrist = if (type == VarselType.FORHANDSVARSEL_STANS_AV_SYKEPENGER) LocalDate.now()
+                    .plusWeeks(3) else null,
```

Comment:
Så hvis varselet er `Unntak` så plukkes det ikke opp av ExpiredVarsel cronjob nå? Ettersom vi ikke setter en frist, og vi har vel en sjekk i den SQLen som henter utgåtte varsler.
(Ser vi har en `NOT EXISTS ( ... AND vurdering.status IN ('UNNTAK', 'OPPFYLT', 'IKKE_AKTUELL')`)

Reply:
Ja, stemmer, den plukker ikke opp varsler uten svarfrist. Laget en test på det også :+1: